react/FilePicker/FilePickerBody.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+
+import { models, useQuery } from 'cozy-client'
+import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
+import LoadMore from 'cozy-ui/transpiled/react/LoadMore'
```

Comment:
No absolute path please. Relative one aka '../../'. With absolute path you break the first watch of the project and also it's subject to issue with reloading. 

We've a babel plugin that convert relative path to absolute once we publish the code.

Reply:
Oops, it's fix 👍